<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cb_mainwindow</class>
 <widget class="QMainWindow" name="cb_mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>927</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Block calendar</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QTimeEdit" name="time_edit_start_of_day"/>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Start of day</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QTimeEdit" name="time_edit_end_of_day"/>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>End of day</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QSpinBox" name="spinbox_free_slots"/>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Requested free slots of 30'</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QSpinBox" name="spinbox_lookahead_days"/>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Nr of days lookahead</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLineEdit" name="line_edit_focus_string">
              <property name="placeholderText">
               <string>cb: focus time</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Focus string</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QCheckBox" name="checkbox_each_hour">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Run each hour</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="pushbutton_block_now">
              <property name="text">
               <string>Run now</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>124</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Empty calendar</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QDateEdit" name="date_edit_start_date"/>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Start date</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QDateEdit" name="date_edit_end_date"/>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>End date</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QTextEdit" name="text_edit_message"/>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Message</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="pushbutton_empty_now">
              <property name="text">
               <string>Empty now</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>41</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="textedit_log">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>4</verstretch>
       </sizepolicy>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>965</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_help"/>
    <addaction name="action_license"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_help">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="action_license">
   <property name="text">
    <string>License</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
